util/options.cc

```diff
@@ -625,13 +625,23 @@ ColumnFamilyOptions* ColumnFamilyOptions::OptimizeLevelStyleCompaction(
   // level style compaction
   compaction_style = kCompactionStyleLevel;
 
+  // choose type of compression to use based on what's supported
+  // (only go for the fastest ones here)
+  CompressionType type;
+  if (CompressionTypeSupported(kSnappyCompression))
+      type = kSnappyCompression;
```

Comment:
Reformatted. BTW, do you run the code through a tool or spot-check on code review? Just curious...